trade-finance-manager-api/src/v1/__mocks__/mock-tfm-facilities.ts

```diff
@@ -0,0 +1,60 @@
+import { Facility, FACILITY_TYPE, TfmFacilityAmendment } from '@ukef/dtfs2-common';
+import { ObjectId } from 'mongodb';
+
+export const MOCK_FACILITY_SNAPSHOT: Facility = {
+  _id: new ObjectId(),
+  dealId: new ObjectId(),
+  type: FACILITY_TYPE.CASH,
+  hasBeenIssued: true,
+  name: 'Test Facility',
+  shouldCoverStartOnSubmission: true,
+  coverStartDate: new Date('2021-01-01'),
+  coverEndDate: new Date('2023-01-01'),
+  issueDate: null,
+  monthsOfCover: null,
+  details: ['Revolving or renewing basis'],
+  detailsOther: '',
+  currency: {
+    id: 'GBP',
+  },
+  value: 50000,
+  coverPercentage: 10,
+  interestPercentage: 5,
+  paymentType: 'Quarterly',
+  createdAt: 1718728159228,
+  updatedAt: 1718728346991,
+  ukefExposure: 5000,
+  guaranteeFee: 4.5,
+  submittedAsIssuedDate: '1718728345615',
+  ukefFacilityId: '0040769518',
+  feeType: 'At maturity',
+  feeFrequency: 'Monthly',
+  dayCountBasis: 360,
+  coverDateConfirmed: true,
+  hasBeenIssuedAndAcknowledged: null,
+  canResubmitIssuedFacilities: null,
+  unissuedToIssuedByMaker: {},
+};
+
+export const MOCK_AMENDMENT: TfmFacilityAmendment = {
```

Comment:
There does seem to be an existing `mock-amendment.js` file - does everythign break if we use this MOCK_AMENDMENT instead (and if not, could that file be replaced by a TS version containing this?)

Reply:
There's also the file `mock-cash-contingent-facilities.js`. Similiarly is it possible to replace one with the other?

Reply:
Good idea - I've changed the mock-amendment.js file to be TS and move the amendment here into that file instead. However replacing `mock-cash-contingent-facilities.js` with a typescript file is turning out to be harder than expected - essentially as it's used in so many places, if I make it a TS file it will change how it's imported across all those files and make this PR a bit messy. So I'm keen to keep it for now in a separate `mock-facility-snapshot` file.

Reply:
Sounds good, happy to leave that